### ISSUE TITLE
Error thrown by mkdir /codewind-workspace/cw-temp/<project>

### ISSUE THREAD

**Issue by makandre** [eclipse/codewind#1723]
**Codewind version:** 0.8.0
**OS:**

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**

**Description:**

I see this error in Codewind log if I create a project using the same name as I project that had already been removed from Codewind.

```
[13/01/20 16:12:06 /portal/routes/projects/remoteBind.route.js] [ERROR] Project creation failed for project apjava01. Error: { [Error: EEXIST: file already exists, mkdir '/codewind-workspace/cw-temp/apjava01']
  errno: -17,
  code: 'EEXIST',
  syscall: 'mkdir',
  path: '/codewind-workspace/cw-temp/apjava01' }
```

I believe it is thrown here: https://github.com/eclipse/codewind/blob/master/src/pfe/portal/routes/projects/remoteBind.route.js#L134

Probably related to https://github.com/eclipse/codewind/pull/1629 since we do not remove the folder in `cw-temp` anymore

**Steps to reproduce:**

**Workaround:**

**Comment by makandre** [comment]
@tobespc FYI

**Comment by stalleyj** [comment]
The behaviour of syncing has changed slightly so we are no longer cleaning up the temp directory. We should add the directory delete to our unbind functionality.

**Comment by jgwest** [comment]
@markcor11: Unable to locate pipeline with name: `Validate`

**Comment by makandre** [comment]
This fixes Codewind projects, but for Appsody it still a problem because `project.workspace` is `/mounted-workspace` for Appsody projects.   The temp dir path needs to be constructed using the `global.codewind.CODEWIND_WORKSPACE` variable.

/reopen

**Comment by jgwest** [comment]
Moving back to In Progress per above.

**Comment by stalleyj** [comment]
PR here https://github.com/eclipse/codewind/pull/2036

**Comment by malincoln** [comment]
@makandre pls verify and close. Thanks

**Comment by makandre** [comment]
Fix #2036 is not in 0.9.0

https://github.com/eclipse/codewind/blob/0.9.0/src/pfe/portal/modules/User.js#L568

**Comment by stalleyj** [comment]
@makandre Issue is not stop-ship so the fix went into master.
